project1/index.html

```diff
@@ -6,67 +6,66 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <title>Smart Octopus</title>
 </head>
 
 <body>
-
-    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
-
-    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
-        intelligent.</h2>
-
-    <figure>
-        <img src="images/merlin.jpg" alt="lignt pink octopus">
-        <figcaption>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
-            uncertain how the trait evolved</figcaption>
+    <h1>Yes,
+        the Octopus Is Smart as Heck. But Why?</h1>
+    <h2>It has eight arms,
+        three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so intelligent.</h2>
+    <figure><img src="images/octopus.jpg" alt="octopus illustration">
```

Comment:
Your `alt` text should replace the image for someone who cannot see it because of impaired vision, a slow internet connection or in the case of a broken image. You should endeavor to write more descriptive or informative `alt` text.